### DISCUSSION TITLE
v1.4 No Longer Export Events?

### DISCUSSION THREAD

**Discussion by DodgeyDan** [blakeblackshear/frigate#12333] [Ask A Question] [answered]
### What is your question:

I've been on 1.4 for a while, and although it seems to have many advantages, version 1.3 I could click an object event and it had an export button to save the event locally.

have I done something wrong or was this removed in 1.4?

**Comment by hawkeye217** [comment]
Exporting is still available. This is discussed in the FAQ in the [release notes](https://github.com/blakeblackshear/frigate/releases).

**Comment by DodgeyDan** [comment]
I guess I should have worded it correctly, I am referring to the ability in 1.3 to simply download that event data in the way you used to use the download button alongside the "Send to Frigate+" Button.

Even if this was an option in the Export window where you can select the time period (last hour etc). 

Just a very quick export me and me only button. That was all I used to use and one of the features I loved.

**Reply by NickM-27** [reply]
right, this can be done. Just right click or long tap on a review item and choose export.

**Reply by hawkeye217** [reply]
> right, this can be done. Just right click or long tap on a review item and choose export.

Yep. What Nick described is exactly what's laid out in the release notes several times.

Are you looking for something else?

**Comment by DodgeyDan** [comment]
Ah I had to right click and show all controls to see it. Is there a way to add a setting in the Settings to "Enable All Controls Always" so this is always visible?

Maybe add the option here as well?
![Screenshot 2024-07-10 054342](https://github.com/blakeblackshear/frigate/assets/4348485/f530c359-cfb9-452a-8ba9-66060bbeab89)

I can see I can enable it in the quick overview up the top too but only on an item by item case.
![Screenshot 2024-07-10 054543](https://github.com/blakeblackshear/frigate/assets/4348485/f606e67c-8654-45ba-8b42-d71790889c44)

**Reply by NickM-27** [reply]
you are still not doing it the way we are describing. Just go to the review screen and right click / long press on an item, it will show in the menu
![Screen Shot 2024-07-09 at 13 50 24 PM](https://github.com/blakeblackshear/frigate/assets/14866235/8db7f54a-4143-4af6-8511-41b4bcdcb473)

**Reply by NickM-27** [reply]
you can also do it by right clicking / long pressing on the events list items

![Screen Shot 2024-07-09 at 13 52 42 PM](https://github.com/blakeblackshear/frigate/assets/14866235/6a68354d-94d2-4a54-997c-2dd84d70aa97)

**Reply by DodgeyDan** [reply, marked as answer]
OK now I get it and understand the reasoning for it. I guess just not so intuitive as it's not in your face the way it used to be.

I would like to see the camera overlay button come back, but so long as I can get there and the feature is not gone its enough.

Thanks,
